src/index.js

```diff
@@ -46,7 +69,7 @@ const mapRoutes = (routes, pathToController) => {
       contr = new handler();
     }
 
-    router.route(myPath)[requestMethod](contr[controllerMethod]);
+    router.route(myPath)[requestMethod](middlewares, contr[controllerMethod]);
```

Comment:
I should dive more into the Router setup, but I guess if the middleware is an empty array the route is  created without middleware and directly calls the controller method, right?